Game/Scripts/PlayerMoveScript.cpp

```diff
@@ -157,21 +157,12 @@ void PlayerMoveScript::Move(float deltaTime)
 			SetPlayerState(PlayerActions::DASHING);
 			componentAudio->PostEvent(AUDIO::SFX::PLAYER::LOCOMOTION::FOOTSTEPS_WALK_STOP);
 			componentAudio->PostEvent(AUDIO::SFX::PLAYER::LOCOMOTION::DASH);
-
-			//if (shiftPressed)
-			//{
-			//	movement /= 2;
-			//}
-			Dash();
 		}
 
-		canDash = false;
-		nextDash = 3000 + static_cast<float>(SDL_GetTicks());
+		nextDash = 1000.0f + static_cast<float>(SDL_GetTicks());
```

Comment:
Does this work? As per [the documentation](https://wiki.libsdl.org/SDL2/SDL_GetTicks):
> Get the number of milliseconds since SDL library initialization.

Which is basically the number of milliseconds since the engine started running. I'm almost positive you should use the `deltaTime` for this:
```suggestion
		nextDash = 1000.0f + deltaTime;
```

Reply:
I've been trying around because simply changing the `SDL_GetTicks()` to `deltaTime` does not work. What is being done with `SDL_GetTicks()` is, as you said, get the miliseconds since SDL library initialization, so in here, `nextDash` will always be maked 1000 miliseconds more than the current SDL miliseconds, and when SDL miliseconds reaches that mark, the line that you commented in the next comment gets triggered. I'm gonna change it so it works with `deltaTime` instead of `SDL_GetTicks()` because it seems more intuitive and we are already using `deltaTime` in other scripts rather than `SDL_GetTicks()`.